py/examples/motor_states_stream/main.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) farm-ng, inc.
+#
+# Licensed under the Amiga Development Kit License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://github.com/farm-ng/amiga-dev-kit/blob/main/LICENSE
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import asyncio
+from typing import List
+
+import grpc
+from farm_ng.canbus import canbus_pb2
+from farm_ng.canbus.canbus_client import CanbusClient
+from farm_ng.canbus.packet import MotorState
+from farm_ng.service import service_pb2
+from farm_ng.service.service_client import ClientConfig
+
+
+class MotorStatesApp:
+    def __init__(self, address: str, canbus_port: int) -> None:
+        self.address: str = address
+        self.canbus_port: int = canbus_port
+
+        self.async_tasks: List[asyncio.Task] = []
+
+    async def app_func(self):
+        # configure the canbus client
+        canbus_config: ClientConfig = ClientConfig(address=self.address, port=self.canbus_port)
+        canbus_client: CanbusClient = CanbusClient(canbus_config)
+
+        # Task(s)
+        self.async_tasks.append(asyncio.ensure_future(self.stream_motors(canbus_client)))
+
+        return await asyncio.gather(*self.async_tasks)
```

Comment:
```suggestion
        await asyncio.gather(self.stream_motors(canbus_clients))
```
This should just work